pulpcore/pulpcore/app/models/publication.py

```diff
@@ -85,7 +85,9 @@ class Distribution(Model):
 
     Relations:
         publisher (models.ForeignKey): The associated publisher.
-            All publications created by the specified publisher will be automatically associated.
+            All publications created by the specified publisher/repo combo will be automatically
+            associated.
```

Comment:
Consider:
"All publications of the repository that are created by the publisher will be automatically associated."